addons/website_event/views/event_templates.xml

```diff
@@ -6,6 +6,9 @@
         <script type="text/javascript" src="/website_event/static/src/js/website_geolocation.js"></script>
```

Comment:
@JKE-be strange enough, my old comment is gone but the question remains: what is your opinion on changing the xml id here ?